api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/MessageProcessorTest.java

```diff
@@ -84,5 +95,35 @@ void shouldThrowIllegalArgumentExceptionForMessageWithNoProcessor() {
         verifyNoMoreInteractions(messageRepository);
     }
 
+    @Test
+    void shouldLogMessageProcessResults() {
+        Message cardMessage = MessageTestDataFactory.aValidMessageWithType(CREATE_NEW_CARD);
+        lenient().when(messageRepository.findAllMessagesByTypeOrderedByDate(CREATE_NEW_CARD)).thenReturn(asList(cardMessage, cardMessage));
+        lenient().when(messageRepository.findAllMessagesByTypeOrderedByDate(MAKE_FIRST_PAYMENT)).thenReturn(emptyList());
+        lenient().when(messageRepository.findAllMessagesByTypeOrderedByDate(SEND_FIRST_EMAIL)).thenReturn(emptyList());
+        //When
+        messageProcessor.processAllMessages();
+        //Then
+        List<ILoggingEvent> events = TestAppender.getEvents();
+        assertThat(events.size()).isEqualTo(1);
```

Comment:
Minor, I prefer to use `assertThat(events).hasSize(1)`.

Reply:
Changed to hasSize.